CHANGELOG.md

```diff
@@ -3,13 +3,15 @@
 ### Changes
 - Support for disabling operation saving based on disk utilization ([#1496](https://github.com/neptune-ai/neptune-client/pull/1496))
 
+### Fixes
```

Comment:
Should be a new version of neptune (possibly 1.8.3 or check current master for unreleased)

Reply:
True! Fixed! :D